src/native/corehost/fxr/fx_ver.cpp

```diff
@@ -70,31 +70,18 @@ bool fx_ver_t::operator >=(const fx_ver_t& b) const
 
 pal::string_t fx_ver_t::as_str() const
 {
-    pal::stringstream_t stream;
-    stream << m_major << _X(".") << m_minor << _X(".") << m_patch;
+    pal::string_t version = pal::to_string(m_major);
```

Comment:
Can we remove  `#include <sstream>` and `#include <locale>` from pal.* files?

Reply:
Some things still use stringstream, so we still need sstream. I think locale can go though.